fidesctl/src/fidesctl/cli/util_comands.py

```diff
@@ -35,14 +35,14 @@ def init(ctx: click.Context, fides_directory_location: str) -> None:
         "cli": {"server_url"},
     }
 
-    print("Initializing Fidesctl...\n")
+    click.echo("Initializing Fidesctl...\n")
```

Comment:
According to [the `click.echo` documentation](https://click.palletsprojects.com/en/7.x/utils/#printing-to-stdout), a newline is emitted by default.
```suggestion
    click.echo("Initializing Fidesctl...")
```